blockchain/contracts/MainContract.sol

```diff
@@ -155,26 +153,26 @@ contract NFTtheWorld {
         dropData[_dropHash].reservedCount += _numberOfNFTs;
         nftReservations[msg.sender][_dropHash] = _numberOfNFTs;
         shuffle(_dropHash);
-        for (uint256 j = 0; j < nftReservations[msg.sender][_dropHash]; j++) {
+        for (uint256 j = nftReservations[msg.sender][_dropHash]; 0 < j; j--) {
```

Comment:
why? :D

Reply:
Because with multiple Reservations for 1 user, otherwise line 158 availableNFTs[_dropHash][j] could go out of index, if for example the highest index would get pop()-ed

Reply:
okay, thanks :)